experimental/framework/fluid-static/src/types.ts

```diff
@@ -96,6 +96,12 @@ export interface IServiceAudience<M extends IMember> extends IEventProvider<ISer
      * Returns the current active user on this client once they are connected. Otherwise, returns undefined.
      */
     getMyself(): M | undefined;
+
+    /**
+     * Gets the member matching the clientId if it is present
+     * @param clientId The clientId to match to a member
+     */
+    getMemberByClientId(clientId: string): IMember | undefined;
```

Comment:
If we return type `M` then each client can still specify their strong-typing for the member returned
```suggestion
    getMemberByClientId(clientId: string): M | undefined;
```
```suggestion
    getMemberByClientId(clientId: string): IMember | undefined;
```

Reply:
good call, missed this since i copy-pasted the signature from serviceaudience